tests/parametric/test_otel_env_vars.py

```diff
@@ -0,0 +1,182 @@
+import pytest, os
+from utils import missing_feature, context, scenarios, features
+
+
+@scenarios.parametric
+@features.open_tracing_api
+class Test_Otel_Env_Vars:
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_SERVICE": "service",
+                "OTEL_SERVICE_NAME": "otel_service",
+                "DD_TRACE_LOG_LEVEL": "error",  # Node uses DD_TRACE_LOG_LEVEL
+                "DD_LOG_LEVEL": "error",
+                "DD_TRACE_DEBUG": "false",
+                "OTEL_LOG_LEVEL": "debug",
+                "DD_TRACE_SAMPLE_RATE": "0.5",
+                "OTEL_TRACES_SAMPLER": "traceidratio",
+                "OTEL_TRACES_SAMPLER_ARG": "0.1",
+                "DD_TRACE_ENABLED": "true",
+                "OTEL_TRACES_EXPORTER": "none",
+                "DD_RUNTIME_METRICS_ENABLED": "true",
+                "OTEL_METRICS_EXPORTER": "none",
+                "DD_TAGS": "foo:bar,baz:qux",
+                "OTEL_RESOURCE_ATTRIBUTES": "foo=otel_bar,baz=otel_qux",
+                "DD_TRACE_PROPAGATION_STYLE_INJECT": "b3,tracecontext",
+                "DD_TRACE_PROPAGATION_STYLE_EXTRACT": "b3,tracecontext",
```

Comment:
Does node support DD_TRACE_PROPAGATION_STYLES? Python is planning to remove `DD_TRACE_PROPAGATION_STYLE_INJECT` and `DD_TRACE_PROPAGATION_STYLE_EXTRACT` in the next release so I would prefer to use `DD_TRACE_PROPAGATION_STYLES` here

```suggestion
                "DD_TRACE_PROPAGATION_STYLE": "b3,tracecontext",
```